include/gridtools/sid/sid_shift_origin.hpp

```diff
@@ -52,12 +52,9 @@ namespace gridtools {
               public:
                 template <class Arg, class Offsets>
                 shifted_sid(Arg &&original_sid, Offsets &&offsets) noexcept
-                    : delegate<Sid>(std::forward<Arg>(original_sid)), m_origin{[this, &offsets]() {
-                          auto &&strides = get_strides(this->m_impl);
-                          ptr_diff_type<Sid> ptr_offset{};
-                          multi_shift(ptr_offset, strides, offsets);
-                          return get_origin(this->m_impl) + ptr_offset;
-                      }()},
+                    : delegate<Sid>(std::forward<Arg>(original_sid)),
+                      m_origin(get_origin(this->m_impl) +
```

Comment:
Looks a bit strangely formatted but I guess that’s clang-format work.

Reply:
yes, clang-fomrat